### PR TITLE
Temporary fix for broken things while the editor is beign moved to modules

### PR DESCRIPTION
**Pull request by sdinesh86**
Many of the current implementations in editor rely on the global editor variable, due to move to modules for editor

### CODE REVIEW

**Comment by Mugen87** [comment]
Can you please list what things are broken now? I think we should find a way to fix them in a more appropriate way than globally exposing `editor`.

File: editor/index.html [hunk 1]
```diff
@@ -67,7 +67,8 @@
 
 			//
 
-			var editor = new Editor();
+            var editor = new Editor();
+            window['editor'] = editor;
```
Comment (Mugen87) [inline editor/index.html]: Please watch for correct formatting.

Comment (sdinesh86) [reply editor/index.html]: will take care of it

**Comment by sdinesh86** [comment]
> Can you please list what things are broken now? I think we should find a way to fix them in a more appropriate way than globally exposing `editor`.

Drag and drop in the outliner is not working,i didnt get time to look into other things but will be looking into them

**Comment by Mugen87** [comment]
> Drag and drop in the outliner is not working,i didnt get time to look into other things but will be looking into them

Solved via #18243. 

Closing this PR since adding `editor` to `window` is not an appropriate solution. Let's try to fix other issues similar to #18243.

**Pull request closed without merging**